ovs/src/main/java/org/dcsa/conformance/standards/ovs/party/OvsFilterParameter.java

```diff
@@ -4,22 +4,34 @@
 
 import java.util.Arrays;
 import java.util.Map;
+import java.util.Set;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+@Getter
 public enum OvsFilterParameter {
   CARRIER_SERVICE_NAME("carrierServiceName"),
   CARRIER_SERVICE_CODE("carrierServiceCode"),
   UNIVERSAL_SERVICE_REFERENCE("universalServiceReference"),
-  VESSEL_IMO_NUMBER("vesselIMONumber"),
-  VESSEL_NAME("vesselName"),
-  CARRIER_VOYAGE_NUMBER("carrierVoyageNumber"),
-  UNIVERSAL_VOYAGE_REFERENCE("universalVoyageReference"),
-  UN_LOCATION_CODE("UNLocationCode"),
-  FACILITY_SMDG_CODE("facilitySMDGCode"),
-  START_DATE("startDate"),
-  END_DATE("endDate"),
-  LIMIT("limit"),
+  VESSEL_IMO_NUMBER("vesselIMONumber", false, "*/vesselSchedules/*/vesselIMONumber"),
+  VESSEL_NAME("vesselName", false, "*/vesselSchedules/*/vesselName"),
+  CARRIER_VOYAGE_NUMBER(
+      "carrierVoyageNumber",
+      false,
+      "*/vesselSchedules/*/transportCalls/*/carrierExportVoyageNumber",
+      "*/vesselSchedules/*/transportCalls/*/carrierImportVoyageNumber"),
+  UNIVERSAL_VOYAGE_REFERENCE(
```

Comment:
Since the USR can be either importVoyageReference or exportVoyageReference, how is this handled in the param matching validation? If one is matching and another not does it throw validation?

Reply:
The code gets the list of carrierExportVoyageNumber , carrierImportVoyageNumber values from the response and tries to match with the query parameter value, if it didnt match then non conformat else its conformant.

Reply:
Does this validates carrierImportVoyage and exportVoyage or universalImportVoyageNumber and universalExportVoyageNumber? As the standard says 
<img width="525" alt="image" src="https://github.com/user-attachments/assets/95d8cc16-5565-4d2b-9ffe-36acf83fc08a">

Also if universalImportVoyageNumber is matched and universalExportVoyageNumber is not matched I think that is a valid case as per the standard. How does this validation catches it?

Reply:
1) If the query parameter contains universalVoyageReference, then the jsonPaths from the OvsFilterParameter is fetched, in this case universalImport/ExportVovageParameter values from the response. These values are matched against the query parameter value. If none is matched then non conformant else conformant.  The same applies to CarrierVoyageNumber as well.
2) We are trying to find the matching values in either of the attribute values. If the query parameter is matched in one attribute value and not in another then its a valid case.

Reply:
Thanks Preetam for the clarification